src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.SparkMaxAbsoluteEncoder;
+import com.revrobotics.CANSparkMax.ControlType;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.config.Config;
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.SwerveSubsystem;
+import java.util.Arrays;
+
+public class ArmCommand extends CommandBase {
+  int level;
+  boolean isTop;
+  double integralMinimum = -0.5;
+  double integralMaximum = 0.5;
+  double z;
+  double nodeX;
+  double armSpeed = 0.2;
+
+  double targetAngularDistanceBottomArm;
+  double targetAngularDistanceTopArm;
+
+  /** Creates a new ArmExtend. */
+  
+  public ArmCommand(int level, boolean isTop) {
+    this.level = level;
+    this.isTop = isTop;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(ArmSubsystem.getInstance());
+  }
+
+
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    ArmSubsystem.getInstance().m_pidControllerBottomArm.setPositionPIDWrappingMaxInput(integralMaximum);
+    ArmSubsystem.getInstance().m_pidControllerBottomArm.setPositionPIDWrappingMinInput(integralMinimum);
+    ArmSubsystem.getInstance().m_pidControllerTopArm.setPositionPIDWrappingMaxInput(integralMaximum);
+    ArmSubsystem.getInstance().m_pidControllerTopArm.setPositionPIDWrappingMinInput(integralMinimum);
```

Comment:
You should reemove these 4 lines of code that deal with PositionPIDWrapping.

I think they are messing with the PID.

PositionPIDWrapping is needed when a mechanism can rotate an infinite amount of rotations (like swerve steering). The arm definitely cannot rotate more than 1 rotation so PositionPIDWrapping is not needed.